internal/extensions.go

```diff
@@ -0,0 +1,63 @@
+package internal
+
+import (
+	"fmt"
+	"github.com/wal-g/wal-g/internal/storages/storage"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"plugin"
+)
+
+var Extensions []Extension
+
+type Extension interface {
+	TryPrintHelp(command, firstArgument string) bool
+	HasCommand(command string) bool
+	Execute(command string, uploader *Uploader, folder storage.Folder, args []string)
+	HasAllowedConfigKey(key string) bool
```

Comment:
Maybe it's better to replace this function with function returning all allowed config keys for this extension.

Reply:
Fixed